Source/WTF/wtf/SystemTracing.h

```diff
@@ -355,22 +355,25 @@ enum WTFOSSignpostType {
 
 #elif USE(SYSPROF_CAPTURE)
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-zero-length"
```

Comment:
Instead of this, let's use the `IGNORE_WARNINGS_{BEGIN,END}` macros defined by WTF:

```suggestion
IGNORE_WARNINGS_BEGIN("format-zero-length")
```